src/reducers.js

```diff
@@ -0,0 +1,43 @@
+import { Record } from 'immutable'
+import _ from 'lodash'
+import { workspaceAccessLevels } from 'src/libs/utils'
+
+
+export const initialState = new (Record({
```

Comment:
Since there is only one store, you need to decide how to organize it. For 'page-based' apps, it's typical to have the top level divided up by page, hence the `workspaces` key here.